src/mavis/util.py

```diff
@@ -390,6 +390,7 @@ def soft_null_cast(value):
             comment='#',
             na_values=['None', 'none', 'N/A', 'n/a', 'null', 'NULL', 'Null', 'nan', '<NA>', 'NaN'],
         )
+        df = df.where(pd.notnull(df), None)
```

Comment:
this still exists in the v3 branch I think? could you make the same PR to it?

Reply:
I thought this is a PR to the v3 branch. The regular develop branch isn't using pandas

Reply:
ah sorry. I was looking at a lot of mavis PRs yday i must have commented on the wrong one